erpnext/patches/v13_0/delete_old_agri_doctypes.py

```diff
@@ -0,0 +1,7 @@
+import frappe
+
+def execute():
+	# delete old Agri Doctypes
+	for doctype in ['Fertilizer', 'Fertilizer Content', 'Crop', 'Crop Cycle', 'Agriculture Task']:
+		if frappe.db.exists("DocType", doctype):
```

Comment:
You should use `frappe.db.table_exists` instead to check for entire tables, rather than records

Reply:
Well the table could exist because frappe won't drop a table on delete_doc causing the patch to fail

Reply:
Will that not cause an issue with the billing flag?

Reply:
That error occurs because the Doctype record exists